http/bit.lua

```diff
@@ -11,7 +11,7 @@ This means we can ignore the differences between bit libraries.
 if _VERSION ~= "Lua 5.1" then
     -- Lua 5.3+ has built-in bit operators, wrap them in a function.
 	-- Use debug.getinfo to get correct file+line numbers for loaded snippet
-	local info = debug.getinfo(1, "Sl")
+	local info = debug and debug.getinfo(1, "Sl") or { currentline = 14 }
```

Comment:
I don't want to have to keep this line number in sync. Would be better to just set to `-1`

Reply:
ok.
`-1` is also the value when using bytecode  without debug info.